h2o-py/h2o/grid/grid_search.py

```diff
@@ -354,6 +354,22 @@ def train(self, x=None, y=None, training_frame=None, offset_column=None, fold_co
         parms["x"] = x
         self.build_model(parms)
 
+    def resume(self, recovery_dir=None):
+        """
+        Resume previously stopped grid training.
+
+        :param recovery_dir: When specified, the grid and all necessary data (frames, models) will be saved to this
```

Comment:
Why'd recovery dir be defined on resume ? Shouldn't that be part of GS initial params ?